HIP 138: subsidy adjustments, involve subnetwork governance (#1126)
helium/HIP: 0138-return-to-hnt.md

## Stakeholders

All Helium Ecosystem participants are affected directly or indirectly by this proposal.

Specifically Helium Security Token (HST) holders are negatively affected by this HIP. HST holders are investors in, and early employees of, Nova Labs, Inc., who currently receive 30% of emitted HNT, decreasing every year to 15%. This HIP will redirect the approximately 11 Million remaining HNT scheduled to be emitted to HST holders directly to the rest of the network's participants.

## Detailed Explanation

This proposal suggests several tokenomic changes to simplify the Helium ecosystem and promote future network development.

At a high level, we propose to:

1. Discontinue HNT emissions to HST holders, and emit all HNT towards network participants directly based on the Utility Score rather than towards the IOT and MOBILE treasuries.
2. Discontinue IOT and MOBILE emissions, subject to veIOT and veMOBILE votes, and reward participants directly in HNT based on the Utility Score, subject to a veHNT vote. 

> 3. Revoke the remaining 18.2B MOBILE currently in the MOBILE Operations Fund, and request that Helium Foundation burn them.

Reviewer: ```suggestion
3. Revoke the remaining 18.2B MOBILE currently in the MOBILE Operations Fund, and request that Helium Foundation burn these tokens.
```

4. Recognize the 4.2 million unissued HNT from Helium L1 after [HIP-20][hip-20] and issue them in two ways:
   1. Establish a new 1.3M HNT MOBILE Growth Fund to fund future development of the MOBILE network, to be administered by Helium Foundation.
   2. Emit the remaining 2.9M HNT directly into the MOBILE treasury on an epochal basis until the next halvening (2025-08-01).

> The existing IOT and MOBILE treasuries remain operational and subject to veHNT governance. Holders of subnetwork tokens may continue to hold them, or may exchange them for HNT at any time through the treasuries. The exchange rate of the MOBILE treasury will continue to improve until the halvening due to the ongoing HNT emissions defined in 4-2 above. 

Reviewer: ```suggestion
The existing IOT and MOBILE treasuries remain operational and subject to veHNT governance. Holders of subnetwork tokens may continue to hold them, or may exchange them for HNT at any time through the treasuries. The exchange rate of the MOBILE treasury will continue to improve until the halvening due to the ongoing HNT emissions defined in 4-2 above.
```

## Implementation

The implementation of this proposal requires independent changes to HNT, IOT and MOBILE, and several votes. We describe these steps below.

### Voting

Because this proposal intends to redirect HNT emissions from HST directly to participants on the network (Hotspot owners, etc.) and revoke a portion of the MOBILE premine that was passed by HIP-51 (HNT voters), it requires a veHNT vote. We propose a simple binary vote for this portion of the HIP. 

In addition, this HIP proposes to halt emissions of IOT and MOBILE tokens, which are subject to veIOT and veMOBILE governance. With the passage of the veHNT vote, HNT emissions will come to replace existing IOT and MOBILE emissions, providing rewards to IOT and MOBILE network participants, while emissions of HNT to the IOT and MOBILE treasuries will cease.

Therefore, veIOT and veMOBILE holders will each vote separately on the proposal to stop the emission of IOT and MOBILE tokens simultaneously with the cessation of HNT emissions to the treasuries and the begin of HNT emissions to the treasuries.

> Note that veIOT or veMOBILE holders could theoretically vote to continue emission of a subnetwork token, contrary to the recommendation of this proposal, but this would result in the issuance of tokens unbacked by treasury HNT, thereby diluting the subnetwork token supply and impairing the treasury exchange rate, and thus the value in HNT terms, of the existing subnetwork tokens.

Reviewer: ```suggestion
Note that veIOT or veMOBILE holders could theoretically vote to continue emission of a subnetwork token, contrary to the recommendation of this proposal, but this would result in the issuance of tokens unbacked by treasury HNT. It's notable that this would dilute the subnetwork token supply and reduce the exchange rate, in HNT terms, of the existing subnetwork tokens.
```

> Should the veHNT vote fail, the veIOT and veMOBILE votes will be void.

Reviewer: ```suggestion
Should the veHNT vote fail, the veIOT and veMOBILE votes will be void. If either subnetwork vote fails, the veHNT decision will stand.
```

### Timeline

> We expect that this proposal, if approved, can be fully implemented on or before January 15,  2025.

Reviewer: ```suggestion
We expect that this proposal, if approved, can be fully implemented on or before January 15, 2025.
```
